### PR TITLE
Add more volunteers to chair group

### PR DESCRIPTION
**Pull request by echeran**
Collecting names of more people who volunteered for chair group to the list.
@DavidFatDavidF @nbouvrette

Anyone else who wants to volunteer can use this PR for convenience and add themselves.

@zbraniecki @stasm

### CODE REVIEW

**Comment by stasm** [comment]
I'd like to volunteer as well. Should I add an inline suggestion to the PR with my name?

**Comment by romulocintra** [comment]
> I'd like to volunteer as well. Should I add an inline suggestion to the PR with my name?

Yes please ...

File: guidelines/chair-group-members.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 - Romulo Cintra
 - Elango Cheran
 - Mihai Nita
+- David Filip
+- Nicolas Bouvrette
```
Comment (stasm) [inline guidelines/chair-group-members.md]:
```suggestion
- Nicolas Bouvrette
- Staś Małolepszy
```

File: guidelines/chair-group-members.md [hunk 2]
```diff
@@ -3,3 +3,6 @@
...
+- Staś Małolepszy
```
Comment (rxaviers) [inline guidelines/chair-group-members.md]: I also would like to volunteer:

```suggestion
- Staś Małolepszy
- Rafael Xavier de Souza
```

**Comment by jamuhl** [comment]
I would love to volunteer - but got a few concerns:

- I already see there a good mix of people in this group (with more knowledge in this space) - so adding more cooks might not be better...idk?
- How will work be organized? I must admit if most of the work has to be done in a synchronous manner I might be more slowing this down than speed it up

So, feel free to add me when closing the call for chair members and you think you still need more people - if not I would be happy to keep my contributions to the work in the wider group.

File: guidelines/chair-group-members.md [hunk 3]
```diff
@@ -6,3 +6,4 @@
 - David Filip
 - Nicolas Bouvrette
 - Staś Małolepszy
+- Rafael Xavier de Souza
```
Comment (aphillips) [inline guidelines/chair-group-members.md]:
```suggestion
- Rafael Xavier de Souza
- Addison Phillips
```

**Comment by echeran** [comment]
If someone can give me an approval, I can merge this PR for chair group volunteers.  

(Even though we are planning on using in-person meetings for getting consensus on decisions, I think merging this PR makes sense since we are all just volunteering.)

**Pull request merged**